docs/upgrade.md

```diff
@@ -87,6 +87,10 @@ process, for example:
 
 # Upgrading to v1.55.0
 
+## Open registration without verification is now disabled by default
+Synapse will refuse to start if registration is enabled without email, captcha, or token-based verification unless the new config 
+flag `enable_registration_without_verification` is set to "true".
+
 ## `synctl` script has been moved
```

Comment:
```suggestion
## Open registration without verification is now disabled by default

Synapse will refuse to start if registration is enabled without email, captcha, or token-based verification unless the new config 
flag `enable_registration_without_verification` is set to "true".


## `synctl` script has been moved
```

Bah, of trivial importance, but I think we usually try and format the markdown so there's one line between a heading and the paragraph, then 2 lines between the paragraph and the next heading. (This is just so that it's still 'nice enough' to read the raw Markdown, which is (was?) kind of the point of Markdown)